### ISSUE TITLE
Can't get json object values

### ISSUE THREAD

**Issue by simaomp** [ufna/VaRest#298]
Hi guys, so I'm trying to get variables with Varest Plugin. I found some problems even knowing some php, mysql and others. I saw some posts here where this code(try3) achieve to a get http request works, unfortunately not for me. UE 4.24.3 Version

This is php file with a json output: https://imgur.com/a/JxFbQgD

And those are what I try to built in Blueprints with no sucess:
Try 1: https://imgur.com/a/J0FeTb3 https://imgur.com/a/yoML5yL 
Output1: https://imgur.com/a/cEqn81l

In try 2 i got these output since html tag to end of json object
Output2: https://imgur.com/a/spz3hOl

Try 3: https://imgur.com/a/Eff88fB 
Output 3: https://imgur.com/a/m1bP95l

Hope someone can help me I spend one day long trying to solve these. I would apreciate a lot, because if this doesn't work i can't move on in the project.

**Comment by simaomp** [comment]
I finally found it. In my db file which have the login to acess databases had html code in there, when i deleted and come with this blueprint.

BluePrint: https://imgur.com/a/6npYRvg
Output: https://imgur.com/a/5dEeOr9
The only code phpfile should have:  echo json_encode(array('variable1' => '$variable1', 'variable2' => $variable2, 'variable3' => $variable3)); 

All works properly know. How can I set the post values instead get? for password and so on, which are the blueprints needed?
